pkg/kubectl/cmd/config/import_configs.go

```diff
@@ -0,0 +1,162 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	clientcmd "k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/util/i18n"
+)
+
+// ImportConfigsOptions contains the assignable options from the args.
+type ImportConfigsOptions struct {
+	configAccess clientcmd.ConfigAccess
+	source       string
+	target       string
+}
+
+var (
+	importConfigsLong = templates.LongDesc(`
+		Imports kubecontext file content
+
+		Adds all cluster, user and context entries of a source kubeconfig
+		into another kubeconfig target file. If target is not provided,
+		the kubeconfig used in path is considered. In case there is a
+		name conflict, i.e. an entry with a given name exists in both
+		files, this entry on the target file is preserved.`)
+
+	importConfigsExample = templates.Examples(`
+		# Imports a kubeconfig file using target as default kubeconfig
+		kubectl config import sourceconfigfile
+
+		# Providing a target file
+		kubectl config import sourceconfigfile --target otherfile`)
+)
+
+// NewCmdConfigImportConfigs creates a command object for the "import" action, which
+// merges two kubeconfig files
+func NewCmdConfigImportConfigs(out io.Writer, configAccess clientcmd.ConfigAccess) *cobra.Command {
+	options := &ImportConfigsOptions{configAccess: configAccess}
+
+	cmd := &cobra.Command{
+		Use:     "import source [(-t|--target)=~/.kube/config]",
+		Short:   i18n.T("Imports kubeconfig file content"),
+		Long:    importConfigsLong,
+		Example: importConfigsExample,
+		Run: func(cmd *cobra.Command, args []string) {
+			cmdutil.CheckErr(options.Complete(cmd, args, out))
+			cmdutil.CheckErr(options.RunImportConfigs(out))
+		},
+	}
+	cmd.Flags().String("target", "", "the file that will receive the entries. Defaults to the regular kubeconfig file")
```

Comment:
If this defaults to the "regular kubeconfig" file, then why is the default a blank string?